starters/gatsby-wordpress-starter/src/templates/postsIndex.jsx

```diff
@@ -46,12 +46,8 @@ export function Head() {
 }
 
 export const pageQuery = graphql`
-	query PostList($offset: Int!, $postsPerPage: Int!) {
-		allWpPost(
-			sort: { fields: [date], order: DESC }
-			limit: $postsPerPage
-			skip: $offset
-		) {
+	query PostList {
```

Comment:
We are already querying all posts at build time in `gatsby-node.js`. You should be able to pass in posts via pageContext down to the postsIndex and pageIndex pages.

In other words, we don't need a pageQuery here at all (even though it's been here awhile I think it can be refactored so we can remove this), similar to how there wasn't one in the pagesIndex.